pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/JavaHeapDumper.java

```diff
@@ -1,60 +1,161 @@
 package org.broadinstitute.ddp.util;
```

Comment:
Is this class expected to be in this PR?
Or do you like my way of concomitant unrelated changes? :)